quote-server/server.js

```diff
@@ -1,32 +1,53 @@
-// server.js
-// This is where your node app starts
-
-//load the 'express' module which makes writing webservers easy
 import express from "express";
-//load the quotes JSON
-import quotes from "./quotes.json" assert { type: "json" };
 
+const quotesApi = "https://api.quotable.io/quotes?page=1"
+const quotes = [];
 const app = express();
-// Now register handlers for some routes:
-//   /                  - Return some helpful welcome info (text)
-//   /quotes            - Should return all quotes (json)
-//   /quotes/random     - Should return ONE quote (json)
-app.get("/", (request, response) => {
-  response.send("Neill's Quote Server!  Ask me for /quotes/random, or /quotes");
-});
+const port = process.env.PORT || 3001;
+
+fetch(quotesApi)
+  .then(response => response.json())
+  .then(data => {
+    data.results.forEach(quote => {
```

Comment:
You could use `forEach` or `map` here - which do you prefer? Why?

Reply:
I prefer using forEach when I don't need to return a new array.